### ISSUE TITLE
Add Sphynx for documentation

### ISSUE THREAD

**Issue by MichaelCurrie** [openworm/open-worm-analysis-toolbox#62]
[Sphinx](http://sphinx-doc.org/) creates lovely and pythonic documentation: e.g. [Pyvot](https://pythonhosted.org/Pyvot/tutorial.html) or [Beautiful Soup](http://www.crummy.com/software/BeautifulSoup/bs4/doc/)

**Comment by eulerkaku** [comment]
This is sort of how the docstrings are showing for a page
![normalizedworm](https://cloud.githubusercontent.com/assets/3817928/6207033/dce2e086-b565-11e4-981c-412e6589476f.png)

**Comment by MichaelCurrie** [comment]
I took a crack at this today.

It seems like to generate HTML from MD, [MkDocs](http://www.mkdocs.org/user-guide/deploying-your-docs/) is used, and to generate HTML from RST, Sphinx should be used.  Sphinx includes a feature called [autodoc](http://sphinx-doc.org/ext/autodoc.html), which is the killer app as far as I'm concerned - it's supposed to rip out the docstrings from each class and function and generate documentation for you.

Both MD and RST can be combined and hosted by ReadTheDocs, which can be configured like Travis-CI can be to run after every GitHub commit.  ReadTheDocs will invoke Sphinx and/or MkDocs, causing the static HTML site to be rebuilt, and then serve it up for you on the web.

To be honest though, our externally-facing API is not going to be that complex (I hope), and it's currently in flux anyway.  And GitHub already lets us have a host a web of Markdown-encoded pages, starting with the README.md of our repository.  And Sphinx to rip out the docstrings looks like it's nontrivial to accomplish.  So I'm going to suggest we avoid creating an elaborate documentation site beyond what's already on GitHub.  That will keep things simpler.

I've refreshed the documentation at http://docs.openworm.org/en/latest/Projects/worm-movement/ as well.
